qutebrowser/completion/completionwidget.py

```diff
@@ -364,15 +364,13 @@ def set_model(self, model):
         old_model = self.model()
         if old_model is not None and model is not old_model:
             old_model.deleteLater()
-            self._selection_model().deleteLater()
```

Comment:
The selection model should be cleared up when the model is: https://code.qt.io/cgit/qt/qtbase.git/tree/src/widgets/itemviews/qabstractitemview.cpp?h=6.7.1#n760